armbotoffboard/subsystems/armsubsystem.py

```diff
@@ -0,0 +1,38 @@
+# Copyright (c) FIRST and other WPILib contributors.
+# Open Source Software; you can modify and/or share it under the terms of
+# the WPILib BSD license file in the root directory of this project.
+
+import sys
+sys.path.append('../armbotoffboard')
```

Comment:
This shouldn't be necessary, please remove it. If it is necessary, we should address whatever it is that is causing it to occur.

Reply:
Which way is more "pythonic"?
I've tried the `__init__.py`, but it still not working.

Reply:
```
[guilherme@archlinux armbotoffboard]$ python3 robot.py sim
Traceback (most recent call last):
  File "/home/guilherme/Projects/examples/armbotoffboard/robot.py", line 11, in <module>
    from robotcontainer import RobotContainer
  File "/home/guilherme/Projects/examples/armbotoffboard/robotcontainer.py", line 9, in <module>
    from subsystems.armsubsystem import ArmSubsystem
  File "/home/guilherme/Projects/examples/armbotoffboard/subsystems/armsubsystem.py", line 6, in <module>
    from armbotoffboard import examplesmartmotorcontroller, constants
ModuleNotFoundError: No module named 'armbotoffboard'
```

Reply:
Use a relative import.

Reply:
thanks, @auscompgeek for the tip :)

The issue was a mistake I made because it isn't necessary to use the `sys` module to import the `ExampleSmartMotorController` and `constants.py`.